contrib/nydusify/pkg/chunkdict/generater/bootstrap.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 Nydus Developers. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package generater
+
+import (
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/dragonflyoss/image-service/contrib/nydusify/pkg/chunkdict/tool"
+)
+
+// Bootstrap in Nydus image
+type Bootstrap struct {
+	BootstrapPath  string
+	NydusImagePath string
+}
+
+func (bootstrap *Bootstrap) Name() string {
+	return "Bootstrap"
+}
+
+func (bootstrap *Bootstrap) Save() error {
+	logrus.Infof("Save chunk and blob table into database")
```

Comment:
Next, we need to call `nydus-image chunkdict generate` to generate the chunk dict image directly, so let's remove the info message.